scripts/cleanup-argo-crds.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+echo "=== Cleaning up Argo CRDs ==="
+
+# Delete any existing workflow resources first
+echo "Removing workflow resources..."
+kubectl delete workflows --all --all-namespaces --ignore-not-found=true || true
+kubectl delete workflowtemplates --all --all-namespaces --ignore-not-found=true || true
+kubectl delete clusterworkflowtemplates --all --ignore-not-found=true || true
+
+# Remove all Argo CRDs
+echo "Removing Argo CRDs..."
+kubectl get crds -o name | grep argoproj.io | xargs -r kubectl delete || true
```

Comment:
One of the issues I ran into with installation was some conflicting argo CRDs at the cluster level, and this ensured things got cleaned up. I do worry a little about the "Removing workflow resources" section above though, as it deletes items across all workspaces and does not filter specifically for argoproj-related items. 

Probably not something we need to spend a lot of time fixing, but a note/warning wherever this script gets documented would be good.